README

```diff
@@ -47,7 +47,8 @@ $ make
 % make install
 
 If you are building from a Git clone rather than a source tarball, you need
-to run ./autogen.sh first.
+to run ./autogen.sh first, after having installed GLib's development files.
```

Comment:
I suggest maybe making it more obvious, you know how nobody reads the README, even less read it carefully, so describing steps that are needed after the steps that follow them is not good.  Instead make two sections, even though they repeat most of it.

Suggest the basic layout:

To make documentation install rst2html, and what happens if its not available

Installation from GIT clone:

Install g* dev (just for my interest why isn't this needed for the tarball?)
autogen.sh (I would suggest only needed first time, but this is the install section not repeated build, so ok)
configure
make
(as root or sudo)
make install

Installing from tarball:

configure
make
(as root or sudo)
make install